src/components/layout/binary-to-deriv/_copyright.js

```diff
@@ -25,6 +25,11 @@ const CopyrightWrapper = styled(Flex)`
     align-items: center;
     justify-content: flex-start;
 
+    p {
+        font-size: var(--text-size-xs);
+        line-height: 1.14;
+    }
```

Comment:
these sizes are deprecated and you should reach for the `Header` component with the `as` prop set to `p` with the `type` set to whichever configuaration required, such as `subtitle-2` for instance.

Reply:
@sean-binary  we can ignore this PR since this is just a reference PR for the current landing page in binary. This shouldn't be merged.